CMakeLists.txt

```diff
@@ -15,26 +15,33 @@ include(cmake/enable_code_coverage_report.cmake)
 # linker flags
 set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--export-dynamic")
 
-include(cmake/add_dependency.cmake)
-add_dependency(ChimeraTK-DeviceAccess 03.00 REQUIRED)
-add_dependency(DOOCS 21.4.0 COMPONENTS server REQUIRED)
-add_dependency(doocs-server-test-helper 01.05 REQUIRED)
+FIND_PACKAGE(ChimeraTK-DeviceAccess 03.08 REQUIRED)
+# we need the zmq client which is part of server component
+FIND_PACKAGE(DOOCS 22.11 REQUIRED COMPONENTS server)
+FIND_PACKAGE(doocs-server-test-helper 01.06 REQUIRED)
+FIND_PACKAGE(Boost REQUIRED COMPONENTS thread system filesystem unit_test_framework)
 
-FIND_PACKAGE(Boost COMPONENTS thread system filesystem unit_test_framework REQUIRED)
 include_directories(${CMAKE_SOURCE_DIR}/include)
 FIND_PACKAGE(PkgConfig REQUIRED)
 
 set(LIBXML++_VERSION "libxml++-2.6")
-PKG_CHECK_MODULES(LibXML++ REQUIRED ${LIBXML++_VERSION})
-PKG_CHECK_MODULES(glib REQUIRED glib-2.0)
-include_directories(SYSTEM "${LibXML++_INCLUDEDIR}/${LIBXML++_VERSION}" ${glib_INCLUDE_DIRS} ${LibXML++_INCLUDE_DIRS})
+PKG_CHECK_MODULES(LibXML++ REQUIRED IMPORTED_TARGET ${LIBXML++_VERSION})
+PKG_CHECK_MODULES(glib REQUIRED IMPORTED_TARGET glib-2.0)
```

Comment:
You can remove that line for glib, this is not necessary - there was probably a bug once in libxml++ which required this, but should work fine without

Reply:
done